.travis.yml

```diff
@@ -64,7 +73,7 @@ before_script:
   - composer update $COMPOSER_FLAGS
 
 script:
-  - SYMFONY_DEPRECATIONS_HELPER=weak vendor/bin/simple-phpunit ${PHPUNIT_FLAGS}
+  - if [[ "$SYMFONY_DEPRECATIONS_HELPER_VALUE" != "" ]]; then SYMFONY_DEPRECATIONS_HELPER=$SYMFONY_DEPRECATIONS_HELPER_VALUE vendor/bin/simple-phpunit ${PHPUNIT_FLAGS}; else SYMFONY_DEPRECATIONS_HELPER=weak vendor/bin/simple-phpunit ${PHPUNIT_FLAGS}; fi;
```

Comment:
we should just specify nothing for the helper, then we get to see the deprecations and can work on eliminating them.

Reply:
The deprecations can be seen after the last commit, mostly the private services are the problem

Reply:
ah, right. great!
i'll add a todo list for the deprecations with some comments. any chance that you can work on those? (not sure how much you are willing to do, after all this just started as a small change during the symfony con hackday ;-) )

Reply:
In 'real world' I'm a web designer and front-end developer, but this should be no problem. I have a minimal to medium knowledge about back-end so it takes more time for me (reading docs, find correct solution, etc.), but if you want I can help in my free time. Anyway I think mostly will be only code refactoring.

Reply:
great oportunity to learn something! and indeed this should be a good intro as its not about building something new but about small refactorings. just pick the ones you are comfortable with to start, and lets get the number of warnings down. hopefully the errors also go down while we do that. i might also do some pull requests against your branch to fix some of the things.